umpf

```diff
@@ -332,12 +332,12 @@ undo_persistent() {
 ### helper ###
 
 nice_branch() {
-	read -r -a reply < <(sed -r -n 's,^(remotes/([^/]*)/|heads/)?(.*),\3 \2,p' <<< "${1}")
+	read -r -a replies < <(sed -r -n 's,^(remotes/([^/]*)/|heads/)?(.*),\3 \2,p' <<< "${1}")
```

Comment:
`nice_branch()` is used in more places. Those need updating as well.

Reply:
Fixed.